gatling/gatling.py

```diff
@@ -125,9 +125,9 @@ def sink(selector, not_done):
         n = 0
         while len(not_done) > 1 or list(selector.get_map()):
             for (fileobj, _, _, p), _ in selector.select():
-                chunk = fileobj.read(4096)
+                chunk = fileobj.read(chunk_size)
```

Comment:
Not sure how we missed this. Not that it matters, 4096 is *the* fastest setting.